fvm_dispatch/src/hash.rs

```diff
@@ -16,9 +17,18 @@ pub trait Hasher {
 pub struct Blake2bSyscall {}
 
 impl Hasher for Blake2bSyscall {
+    // fvm_sdk syscalls only work for WASM targets
+    #[cfg(target_family = "wasm")]
```

Comment:
Do we still need this? I confess I don't fully understand how it will work. What will happen when someone wants to use some new FVM integration test thing that runs purely native (no WASM)?

Reply:
For the moment we need it, to avoid unresolved externs (part of the syscalls within the SDK). We might be able to get rid of it if using the FVM integration test library, I haven't looked closely at that code yet. I've changed this (and the method call one in `message.rs`) to use `unimplemented!()` which will panic if they're ever called instead of returning empty value or error results.

Added an issue to address it later: https://github.com/helix-onchain/filecoin/issues/43